README.md

```diff
@@ -77,14 +77,9 @@ terraform destroy
 
 ![image](https://github.com/adamski201/Railway-Tracker/assets/84942651/60dab6de-ab46-4a31-8076-2c2c07dfc0b2)
 
-
-
-
-
-
 ### Cloud architecture diagram
 
-![Cloud architecture diagram](...)
+![Cloud architecture diagram](https://github.com/adamski201/Railway-Tracker/blob/update-arch-diagram/images/architecture_diagram.png)
```

Comment:
I think there's arguments for putting the S3 bucket in both 'archiving process' and 'report process'.
The S3 bucket has two uses: storing archived data and storing summary reports. 
The summary reports use the archived data, making that the centre of the S3 bucket, which I could see being a reason for putting the S3 in the 'archiving process'. 
Although, the 'report process' interacts with the S3 bucket more than the 'archiving process' - it both uploads and reads from the S3 bucket. I'm not sure what it looks like otherwise, but the fact there are two arrows connecting to the S3 bucket inside of the 'report process' might mean it's better to stick with the S3 bucket inside of the 'report process' for an easier-to-read architecture diagram.

Those are just my thoughts.

Reply:
Thanks guys! 

Agreed @SaniyaShaikhh - thats an oversight on my end! I'll change that now

Also @adamski201 good point you've made. Following on from @isaacschaessenscoleman, Dan suggested to make the S3 outside both just like the database. I believe thats a fair point since its not tied to one process. If everyones happy with that, I'll  update and push the changes